src/types/config.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "networks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "type": "string"
+                    },
+                    "rpc": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "deploymentBlock": {
+                        "type": "integer"
+                    },
+                    "watchdogTimeout": {
+                        "type": "integer"
+                    },
+                    "orderBookApi": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "pageSize": {
+                        "type": "integer"
+                    },
+                    "filterPolicy": {
+                        "type": "object",
+                        "properties": {
+                            "defaultAction": {
+                                "type": "string",
+                                "enum": [
+                                    "ACCEPT",
+                                    "DROP",
+                                    "SKIP"
+                                ]
```

Comment:
Nitpick: create a definition for this type/enum to avoid repetition

Reply:
Nice, done!